templates/macro/_picture_element.html.twig

```diff
@@ -0,0 +1,8 @@
+{% macro picture_element(image_title, xl, lg, md, xs) %}
```

Comment:
maybe reverse the order of the images so it goes from small to big (mobile first).